headers/types/common/common.h

```diff
@@ -6,6 +6,7 @@
 #include "enums.h"
 #include "../dungeon_mode/dungeon_mode_common.h"
 #include "file_io.h"
+#include "../files/wan.h"
```

Comment:
Did you mean to include `../files/files.h`? Otherwise `files.h` is unused and I think there's no point in creating it.

Reply:
Hmmm... I put it cause I already had created for other uncommited file, but it indeed is useless as of now. I’ll remove it.

Reply:
Done